docs/src/screens/gallery/samples/basic.js

```diff
@@ -0,0 +1,34 @@
+const code = context => `
```

Comment:
Here's a good example of how we use the same code across the "thumbnail" and live preview but adjust the CSS to account for different sized containers in those cases. This `context` parameter will be either `gallery-preview` or `gallery`. The on L20 below we check that parameter to apply different classnames inline into the code used by `react-live`.